Arithmetic.pp

```diff
@@ -49,6 +49,7 @@
 %token  minus     \-|−
 %token  times     \*|×
 %token  div       /|÷
+%token  indices   \^
```

Comment:
Caret is a common symbol for the exponent operation. I believe `pow` is implementation specific to the programming languages themselves.